### PR TITLE
added Dealer class, implemented score checking, winning

### PR DESCRIPTION
**Pull request by IStallcup**
Initial Dealer PR :)

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -35,6 +41,25 @@ public void buildDeck()
...
+
+	public void whoWon()
+	{
+		int dealerScore = dealer.calcScore(dealer.hand);
+		int userScore = user.calcScore(user.hand);
+		if (dealerScore > userScore)
+			dealerWon = true;
+		else userWon = true;
+	}
+
```
Comment (leel8on) [inline src/main/java/models/Game.java:89]: What about a push (tie)?

Comment (IStallcup) [inline src/main/java/models/Game.java:89]: Good point! :+1:

File: src/main/java/models/Dealer.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package models;
+
+public class Dealer extends Player{
+
+    public Dealer()
+    {
+
```
Comment (kannabanana) [inline src/main/java/models/Dealer.java:7]: Looks a little empty ;)

File: src/main/java/models/Game.java [hunk 3]
```diff
@@ -35,6 +41,41 @@ public void buildDeck()
...
+	public void dealCardToDealer()
+	{
+		dealer.hand.add(dealCard());
+	}
+
+	public void whoWon()
+	{
+		int dealerScore = dealer.calcScore(dealer.hand);
+		int userScore = user.calcScore(user.hand);
+		if (dealerScore > userScore)
```
Comment (athai) [inline src/main/java/models/Game.java:84]: You're not checking for busting

Comment (IStallcup) [inline src/main/java/models/Game.java:84]: Ooh good point

Comment (leel8on) [inline src/main/java/models/Game.java:84]: Seems like something we need to check for every time a card is dealt. If we did that and set the score to zero on bust, we could keep this logic.

Comment (IStallcup) [inline src/main/java/models/Game.java:84]: Check out new code, did basically what you said minus adding the score set to zero. Do you think I need to change the new bust code?

Comment (leel8on) [inline src/main/java/models/Game.java:84]: i think we can work with that

**Pull request merged**